vertx-web-api-contract/src/main/java/io/vertx/ext/web/api/contract/RouterFactoryOptions.java

```diff
@@ -67,6 +72,7 @@
   private boolean mountResponseContentTypeHandler;
   private BodyHandler bodyHandler;
   private List<Handler<RoutingContext>> globalHandlers;
+  private String exposeConfigurationKey;
```

Comment:
I think we need a more intuitive field name for this config entry, What do you think about operationModelKey?